src/scenes/everest.lua

```diff
@@ -66,7 +68,52 @@ Use the latest ]], { 0.3, 0.8, 0.5, 1 }, "stable", { 1, 1, 1, 1 }, " or ", { 0.8
             clip = false,
             cacheable = false
         }):with(uiu.styleDeep), function()
-            scene.install()
+            -- Check for XNA installed
+            local xnaInstalled =
+                registry.getKey([[HKLM\SOFTWARE\WOW6432Node\Microsoft\XNA\Framework\v4.0\Installed]]) or
+                registry.getKey([[HKLM\SOFTWARE\Microsoft\XNA\Framework\v4.0\Installed]])
```

Comment:
This should preferably check for the operating system _before_ trying to load `registry` and getting keys from it.